gensrc/thrift/HeartbeatService.thrift

```diff
@@ -50,6 +50,7 @@ struct TBackendInfo {
     6: optional i64 be_start_time
     7: optional string be_node_role
     8: optional bool is_shutdown
+    9: optional i64 be_process_uuid
```

Comment:
uuid is  int128, int64 could not be a uuid

Reply:
Rename be_process_uuid to be_process_epoch? Maybe use epoch is enough.